frontend/routes/viewer/components/pinButton/pinButton.component.tsx

```diff
@@ -0,0 +1,109 @@
+/**
+ *  Copyright (C) 2019 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import * as React from 'react';
+import { PinIcon, LabelButton, Container } from './pinButton.styles';
```

Comment:
Try to keep the proper order of imports - the closest paths at the bottom.

Reply:
Sorry, the imports are generated automatically by Visual Studio Code. 
Can you provide what would be the most desired order in this case?

Reply:
```suggestion
import * as React from 'react';
import { Viewer } from '../../../../services/viewer/viewer';
import { VIEWER_EVENTS, NEW_PIN_ID } from '../../../../constants/viewer';
import { PIN_COLORS } from '../../../../styles';
import { PinIcon, LabelButton, Container } from './pinButton.styles';
```